01/index.html

```diff
@@ -9,5 +9,6 @@
 </head>
 <body>
     <p>GitHub też jest fajny!</p>
+    <h1>Git też jest fajny </h1>
```

Comment:
Mam nadzieje, że taki pozostanie :P

PS. Zwróć uwagę, że to komentarz do jedne linii kodu :)